subprojects/gradle/cd/src/main/kotlin/com/avito/ci/BuildStepListExtension.kt

```diff
@@ -64,8 +64,8 @@ open class BuildStepListExtension(
         registerFactory(FlakyReportStep::class.java) { name ->
             FlakyReportStep(buildStepListName, name)
         }
-        registerFactory(DetektCheck::class.java) { name ->
-            DetektCheck(buildStepListName, name)
```

Comment:
Detekt check is not needed anymore. It has been replaced by a `customTask` build step:

```kotlin
register("fastCheck") {
        customTask("detekt") {
            tasksPredicate = TasksPredicate.ByName("detektAll")
        }
}
```